README.md

```diff
@@ -7,3 +7,43 @@ No gui. Candidate are showed in text area. Cuz i really can not find a working n
 Now everything is hardcoded, but i except it can be configured like dwm. It is overkill to add IPC or dynamic config in such a project, though, i am writing it with modularity in my mind.
 
 It will not be too hard to add new input backend engine, enable it by pkgconf check and ifdef at compilation time.
+
+## Install
+### Dependencies
+
+1. Make use you have [rime](https://github.com/rime/librime) installed
+2. Place your desired rime configuration files in ~/.config/wlpinyin/
+    - [Rime's minimal example](https://github.com/rime/librime/tree/master/data/minimal) is a good place to start.
```

Comment:
I want to mention that [brise](https://github.com/rime/brise) may be needed for bootstrap. You don't really need to config manually the first time starting.

I am not sure. I remembered that I don't need to config manually the last time of testing.

Reply:
I am personally unfamiliar with brise, it looks like it is a system to automate the configuration of rime, but it looks depricated.
In other words, it seems like you do not stricly need brise, I will add a section to patch 1, you say if you are happy with it or not :)

Reply:
Yes, you don't strictly need brise. It is just common files that will copied into your local config directory.

Reply:
I think it is fine not to mention that. If you can compile it, you should be able to config rime by yourself.